src/components/structures/RoomView.js

```diff
@@ -1279,12 +1292,7 @@ module.exports = React.createClass({
         }
 
         var messagePanel = (
-            <TimelinePanel ref={(r) => {
-                    this.refs.messagePanel = r;
```

Comment:
This has arguably been broken for a while, but it suddenly became necessary as the messages about the auxpanel getting resized weren't getting through because `this.refs.messagePanel` ended up being null at the critical moment.